src/map/npc.c

```diff
@@ -4813,8 +4860,10 @@ int npc_reload(void) {
 	for (bl = mapit->first(iter); mapit->exists(iter); bl = mapit->next(iter)) {
 		switch(bl->type) {
 			case BL_NPC:
-				if( bl->id != npc->fake_nd->bl.id )// don't remove fake_nd
+				if (bl->id != npc->fake_nd->bl.id ) {// don't remove fake_nd
 					npc->unload(BL_UCAST(BL_NPC, bl), false);
+					status->change_clear(bl, 1);
```

Comment:
Clan Emblems are held by Status Change, so I had to add it in npc_data, I made it last month and if I'm not wrong it is needed to free the sc data. But I'll test/check it ><